handbook/company/communications.md

```diff
@@ -154,31 +143,54 @@ Team members can request reimbursement through [Gusto]([https://app.gusto.com/ex
 Operations will review the expense and reach out to the team member if they have any questions. The reimbursement will be added to the team member's next payroll when an expense is approved.
 >Pilot handles reimbursements differently depending on if the international team member is classified as an employee or a contractor. If the reimbursement is for a contractor, Operations will need to add the expense reimbursement to an upcoming recurring payment or schedule the reimbursement as an off-cycle payment. If the reimbursement is for an employee, no other action is needed; Pilot will add the reimbursement to the team member's next payroll.  -->
 
-### Individualized expenses
-Recurring expenses related to a particular team member, such as coworking fees, are called _individualized expenses_.  These expenses are still considered [non-personnel expenses](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278), with a few extra considerations:
+### Brex
+All Fleeties will be provided with a Brex account and virtual Brex card when they onboard at Fleet. Fleeties will be prompted to order a physical Brex card when they log in to their account. Fleet recommends you order a physical card as soon as possible.
```

Comment:
> Fleeties will be prompted to order a physical Brex card when they log in to their account. Fleet recommends you order a physical card as soon as possible.

^ Both of these sentences can be removed

Only Fleeties who travel need a physical card, and can order one ahead of their first travel: https://fleetdm.com/handbook/company/communications#spending-company-money-while-traveling

(This minimizes the risk of having extra cards floating around, and minimizes mistakes and the need for reimbursements due to accidents.  Fleeties who are unable to get physical cards ahead of their travel can instead get help from other Fleeties as needed, or buy their own food for that trip  (I do it all the time).)

Reply:
Looks like this change from 2023-08-26 might have gotten overlooked: https://github.com/fleetdm/fleet/pull/13466/files#r1306587656